DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -155,28 +174,29 @@ public string RegisterDelegateByCentre(DelegateRegistrationModel delegateRegistr
             }
 
             var supervisorDelegateRecordIdsMatchingDelegate =
-                GetPendingSupervisorDelegateIdsMatchingDelegate(delegateRegistrationModel);
+                GetPendingSupervisorDelegateIdsMatchingDelegate(delegateRegistrationModel).ToList();
 
-            UpdateSupervisorDelegateRecordsWithNewCandidateId(
+            var delegateUser = userDataService.GetDelegateUserByCandidateNumber(
                 candidateNumber,
-                delegateRegistrationModel.Centre,
-                supervisorDelegateRecordIdsMatchingDelegate
+                delegateRegistrationModel.Centre
             );
 
-            return candidateNumber;
-        }
-
-        private void UpdateSupervisorDelegateRecordsWithNewCandidateId(

```

Comment:
We've lost this method resulting in duplicate code in the 2 register delegate methods, is it not possible to keep this?
I suspect we'll end up in the situation where we have even more common code between the 2 methods after I've done some of the refactoring I was planning to do.

Reply:
I'll see where the other conversation goes before committing to anything on this.